strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/metadata/MavenMetadataManager.java

```diff
@@ -108,15 +107,11 @@ public Metadata readMetadata(InputStream is)
     {
         Metadata metadata;
 
-        try
+        try (InputStream inputStream = is)
```

Comment:
I don't think this makes sense as it is. I believe the original stream should be defined here instead.

Reply:
By original stream you mean to directly use the method's parameter `is`- from `readMetadata(InputStream is)`?

To close a resource, try-with-resources requires this resource to be created inside of try-with-resources parenthesis.
That is because of this that I declared a new inputstream variable in try `(InputStream inputStream = is)`.
Passing existing resources varibles directly into try-with-resources is supported from Java 9 on.
Since strongbox now uses Java 8, using original stream is not possible in current configuration.

Reply:
Oh, I didn't realize this was a method parameter.